docs/infrastructure.md

```diff
@@ -0,0 +1,62 @@
+# Infrastructure
+
+This app is deployed on [GOV.UK PAAS][government-paas].
+
+## Space for the app
+
+You may be given a space by your organisation manager, or need to create one
+for the application.
+
+At the time of writing this app runs on the `govuk_development` organisation
+in the `sandbox` space
+
+## Push the app to the PAAS
+
+As we've not set everything up we don't want to start the app now.
+
+```
+$ cf push pact-broker --no-start
+```
+
+we can pass a `-n HOSTNAME` argument here to specify the hostname, otherwise
+it will use the default one of `pact-broker` (as specified in
+[manifest.yml](manifest.yml))
+
+## Create a database
+
+There's a number of different database plans available, but the "Free" one
+should be sufficient:
+
+```
+$ cf create-service postgres Free pact-broker-db
+```
+
+This will take a few minutes. Once completed it can be bound to the application:
+
+```
+$ cf bind-service pact-broker pact-broker-db
+```
+
+## Set up environment variables for credentials
+
+This application requires basic auth credentials of `AUTH_USERNAME` and
+`AUTH_PASSWORD`.
+
+```
+$ cf set-env pact-broker AUTH_USERNAME username
+$ cf set-env pact-broker AUTH_PASSWORD password
+```
+
+## Start the app
+
+```
+$ cf start pact-broker
+```
+
+And then you'll probably want it to run on 2 instances:
+
+```
+$ cf scale pact-broker -i 2
+```
+
+[government-paas]: https://docs.cloud.service.gov.uk/
```

Comment:
This guide seems to be a bit of a "how do you use the PAAS" (which I actually wrote must be 5 years ago now). I've no idea if it's still up to date. I wonder if it's best to just delete the file and if people need to work out any of these things they can use the PAAS docs (which are linked to in the deployment file)

Reply:
Most of it looks OK to me. I do agree that we shouldn't have such a detailed document when a broader knowledge of GOV.UK on PaaS is really what's needed, but there's nothing like that in the dev docs at the moment - just a scattering of (often vague) references to our use of PaaS. For the time being and given our niche use of PaaS, I think it's more helpful to keep this document in case someone needs to work on this repo.

Reply:
I've suggested a compromise: https://github.com/alphagov/govuk-pact-broker/pull/123#discussion_r863536956

Reply:
Seems ok to leave it. I still have low faith in its accuracy.